web/pdf_viewer.js

```diff
@@ -757,7 +757,8 @@ var PDFViewer = (function pdfViewer() {
       return new AnnotationLayerBuilder({
         pageDiv: pageDiv,
         pdfPage: pdfPage,
-        linkService: this.linkService
+        linkService: this.linkService,
+        downloadManager: new DownloadManager()
```

Comment:
I'm not too sure that this will work with the viewer components.

You probably need to add `PDFJS.DownloadManager = DownloadManager;` to https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.component.js, and pass in a `new DownloadManager` when initializing `PDFViewer`, instead of initializing it here?